well_architected_framework/operational_excellence/ops04.sp

```diff
@@ -32,7 +32,7 @@ benchmark "well_architected_framework_ops04_bp01" {
 
   tags = merge(local.well_architected_framework_ops04_common_tags, {
     choice_id = "ops_telemetry_application_telemetry"
-    severity  = "high"
+    risk  = "high"
```

Comment:
@khushboo9024 Is the spacing aligned with `choice_id` above? If not, can you please align them?